ci/docker/os-image-stemcell-builder/scripts/update.sh

```diff
@@ -69,6 +69,9 @@ apt-get -y install qemu-utils
 # CentOS building requirements
 apt-get -y install yum
 
+# Oracle image building requirements
+apt-get -y dosfstools gdisk
```

Comment:
Missing command install.  Should be `apt-get -y install dosfstools gdisk`

Reply:
nice catch. I have not had a chance to try the image build yet actually - I'll give it a go once our office move settles a bit.